wcfsetup/install/files/lib/system/language/preload/RegisterPreloadPhrases.class.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace wcf\system\language\preload;
```

Comment:
```suggestion
namespace wcf\system\language\preload\event;
```

Events are grouped into the `event` namespace of an appropriate “subsystem”.